FAQ.md

```diff
@@ -1,30 +1,32 @@
+For the most part, we don't think it means large changes. We think that the text does a really good job describing the way the TC39 community already conducts itself. We expect that most people will simply continue to behave in the awesome way they have for years.
 
-However, we do expect that people will abide by the spirit and words of the CoC when in "official" JS Foundation spaces. This code has been adopted by the JS Foundation and all of its projects. That means that it'll apply both in community spaces _and_ at JS Foundation events.
+However, we do expect that people will abide by the spirit and words of the CoC when in "official" TC39 spaces. This code has been adopted by TC39. That means that it'll apply both in community spaces _and_ at TC39 events.
 
-In practice, this means the various JS Foundation IRC channels (#jquery, #jquery-dev, etc.), bug tracking and code review tools, and "official" JS Foundation events such as sprints. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+In practice, this means the various TC39 IRC channels, bug tracking and code review tools, and "official" TC39 events. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
 
-### What about JS Foundation events?
+### What about TC39 events?
 
-This Code of Conduct also covers all [JS Foundation events][] since it is all about how we interact as a community. It's about saying that the JS Foundation community will be open, friendly, and welcoming. The core issue is about ensuring the conversations we have are productive and inviting for all.
+This Code of Conduct also covers all TC39 events since it is all about how we interact as a community. It's about saying that the TC39 community will be open, friendly, and welcoming. The core issue is about ensuring the conversations we have are productive and inviting for all.
 
-Real-life events, however, require a bit more care. The JS Foundation wants to be sure that all of its events have policies and procedures in place for handling harassment. It's especially important to us that real-life events take steps to protect the physical and mental security of their participants.
+Real-life events, however, require a bit more care. TC39 wants to be sure that all of its events have policies and procedures in place for handling harassment. It's especially important to us that real-life events take steps to protect the physical and mental security of their participants.
 
-So the JS Foundation will require that all events have some sort of anti-harassment policy in place. The JS Foundation thinks the [Ada Initiative's template][] is pretty good, but we're open to alternatives.
+So TC39 will require that all events have some sort of anti-harassment policy in place. TC39 thinks the [Ada Initiative's template][] is pretty good, but we're open to alternatives.
```

Comment:
> TC39 thinks the [Ada Initiative's template][] is pretty good, but we're open to alternatives.

I believe we should remove this suggestion, saying what TC39 think will create bikeshedding.